src/Resources/config/doctrine/Model.SettingModel.orm.yml

```diff
@@ -21,6 +21,11 @@ Helis\SettingsManagerBundle\Model\SettingModel:
             name: data
             type: json_array
             nullable: false
+        choices:
+            name: choices
+            type: json_array
```

Comment:
`json_array` is deprecated, use `json`
https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/types.html#json-array

Reply:
Fixed for keys `choices` but also `data`